modules/siddhi-launcher/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.wso2.siddhi.sdk</groupId>
         <artifactId>siddhi-sdk-parent</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <version>1.0.0</version>
```

Comment:
@RAVEENSR , Master branch needs to have a SNAPSHOT version. When releasing the component, we can set a release version (non-snapshot version). Please change back to 1.0.0-SNAPSHOT.